source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -129,6 +240,109 @@ This means that in order to have a VM connected to both ipv4 and ipv6 subnets,
 it needs to connect to 2 different networks (through 2 different logical ports),
 one with an IPv4, other with an IPv6 subnet defined on top.
 
+### Networking API entity provisioning
+
+Below you can find the steps required to have dynamic IPv6 addresses on the
+guest VMs, along with the available methods of provisioning it.
+
+1. create a logical network
+    - oVirt UI
+    - oVirt REST API
+    - manageIQ UI
+    - ovirt-provider-ovn REST API. [REST example](#create-a-logical-network)
+2. add a subnet on top
+    - oVirt UI - *requires changes*
```

Comment:
If ovirt-provider-ovn would use dhcp-stateless as default, would be still changes required?

Reply:
changes are not required, unless we want that configurable.